dashboard/src/components/Config/PkgRepoList/PkgRepoList.tsx

```diff
@@ -94,35 +91,37 @@ function AppRepoList() {
       ?.catch(() => setCanEditGlobalRepos(false));
   }, [cluster, kubeappsCluster, kubeappsNamespace, globalReposNamespace]);
 
-  const globalRepos: IAppRepository[] = [];
-  const namespacedRepos: IAppRepository[] = [];
+  const globalRepos: PackageRepositorySummary[] = [];
+  const namespacedRepos: PackageRepositorySummary[] = [];
   repos.forEach(repo => {
-    repo.metadata.namespace === globalReposNamespace
-      ? globalRepos.push(repo)
-      : namespacedRepos.push(repo);
+    if (!repo.namespaceScoped) {
+      globalRepos.push(repo);
+      // ensure listed namespaced repos are those in the current namespace
+    } else if (allNS || repo.packageRepoRef?.context?.namespace === namespace) {
+      namespacedRepos.push(repo);
+    }
   });
 
   const tableColumns = [
     { accessor: "name", Header: "Name" },
     { accessor: "url", Header: "URL" },
-    { accessor: "plugin", Header: "Package Type" },
+    { accessor: "packageFormat", Header: "Package Format" },
     { accessor: "accessLevel", Header: "Access Level" },
     { accessor: "namespace", Header: "Namespace" },
     { accessor: "status", Header: "Status" },
     { accessor: "actions", Header: "Actions" },
   ];
-  const getTableData = (targetRepos: IAppRepository[], disableControls: boolean) => {
+  const getTableData = (targetRepos: PackageRepositorySummary[], disableControls: boolean) => {
```

Comment:
At some point we discussed the alternative of having a single table for all repositories, differentiating the global ones with some distinctive icon (e.g. world icon). Is this something that could be done with the new repos API? (not for this PR/issue of course).